app/controllers/anonymous_feedback/explore_controller.rb

```diff
@@ -9,13 +9,21 @@ def new
     @organisations_list = support_api.organisations_list.map do |org|
       [organisation_title(org), org["slug"]]
     end
+    @explore_by_document_type = Support::Requests::Anonymous::ExploreByDocumentType.new
+    @document_type_list = support_api.document_type_list.to_a.map do |document_type|
+      [document_type.titleize, document_type]
+    end
```

Comment:
You're just following the established pattern for orgs, but I wonder if it would be a good refactor to move this .to_a.map into a helper or something because we use it in two places and it'd be a shame if one got out of sync.  And to do the same for the org list too.